hathor/indexes/partial_rocksdb_tips_index.py

```diff
@@ -127,9 +139,18 @@ def _from_key(self, key: bytes) -> Interval:
         assert len(tx_id) == 32
         return Interval(_from_db_value(begin), _from_db_value(end), tx_id)
 
-    def init_start(self) -> None:
+    def init_start(self, indexes_manager: 'IndexesManager') -> None:
         log = self.log.new(index=f'tips-{self._name}')
-        total = int(self._db.get_property(b'rocksdb.estimate-num-keys', self._cf))
+        total: Optional[int]
+        if self is indexes_manager.all_tips:
+            total = indexes_manager.info.get_tx_count() + indexes_manager.info.get_block_count()
+        elif self is indexes_manager.block_tips:
```

Comment:
I didn't get it. Why not sum `block_tips.get_block_count() + tx_tips.get_tx_count()`? Why not move this logic to the `indexes_manager`?

Reply:
Ok. I agree that we should improve it on a separate refactor.